aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+  description: 'Number of namespaces in an Aerospike cluster.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        fixedColor: 'green',
+        mode: 'fixed',
+      },
+      custom: {
+        align: 'left',
+        cellOptions: {
+          type: 'auto',
+        },
+        inspect: false,
+      },
+      links: [],
+      mappings: [],
+      max: 32,
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Value',
+        },
+        properties: [
+          {
+            id: 'custom.hidden',
+            value: true,
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Time',
+        },
+        properties: [
+          {
+            id: 'custom.hidden',
+            value: true,
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'instance',
+        },
+        properties: [
+          {
+            id: 'custom.filterable',
+            value: true,
+          },
+          {
+            id: 'links',
+            value: [
+              {
+                title: 'Instance overview',
+                url: '/d/aerospike-instance-overview',
+              },
+            ],
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'aerospike_cluster',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'Aerospike cluster',
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'ns',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'namespace',
+          },
+        ],
+      },
+    ],
+  },
+  links: [
+    {
+      title: 'Namespace overview',
+      url: '/d/aerospike-namespace-overview',
+    },
+  ],
+  options: {
+    cellHeight: 'md',
+    footer: {
+      countRows: false,
+      fields: '',
+      reducer: [
+        'sum',
+      ],
+      show: false,
+    },
+    showHeader: true,
+    sortBy: [],
+  },
+  pluginVersion: '10.2.0-59542pre',
+  transformations: [
+    {
+      id: 'sortBy',
+      options: {
+        fields: {},
+        sort: [
+          {
+            desc: true,
+            field: 'Time',
+          },
+        ],
+      },
+    },
+  ],
+};
+
+local unavailablePartitionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(job, aerospike_cluster) (aerospike_namespace_unavailable_partitions{job=~"$job", aerospike_cluster=~"$aerospike_cluster"})',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'bargauge',
```

Comment:
seems like this would be better as a horizontal bar gauge in case there were multiples showing (text for each wouldn't be bumping into each other)